Resources/Textures/DeltaV/Mobs/Customization/Moth/moth_wings.rsi/meta.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
- 4 space indent
- dont mix tabs and spaces
- file needs newline at the end

Reply:
hopefully corrected now

Reply:
change it to 4 spaces per indent instead of 2 and its good

Reply:
should be good now?